client/js/authentication/logout-button.js

```diff
@@ -1,37 +1,33 @@
 import React from 'react'
 import injectSheet from 'react-jss'
 
-import Spacer from '../components/spacer'
 import { logoutUser } from './action-creators'
 
 const LogoutButton = ({ classes }) => (
 	<div>
-		<Spacer height="10px" />
-		<button className={classes.button} onClick={logoutUser}>
-			Logout
-		</button>
+		<div className={classes.button}>
+			<button className={classes.logoutButton} onClick={logoutUser}>
+				Logout
+			</button>
+		</div>
 	</div>
 )
 
 const styles = {
 	button: {
-		color: 'rgb(255, 255, 255)',
-		cursor: 'pointer',
-		height: '63.875px',
-		'min-height': '0px',
-		'text-decoration': 'none solid rgb(255, 255, 255)',
-		'text-transform': 'uppercase',
-		width: '110px',
-		'column-rule-color': 'rgb(255, 255, 255)',
-		'perspective-origin': '43.0312px 31.9375px',
-		'transform-origin': '43.0312px 31.9375px',
-		'caret-color': 'rgb(255, 255, 255)',
-		background: 'rgb(76, 105, 186) none repeat scroll 0% 0% / auto padding-box border-box',
-		border: '1px solid rgb(76, 105, 186)',
-		font: 'normal normal bold normal 13.9504px / normal Helvetica, sans-serif',
-		outline: 'rgb(255, 255, 255) none 0px',
-		padding: '14.938px 19.938px',
-		transition: 'background-color 0.3s ease 0s, border-color 0.3s ease 0s',
+		'padding-left': '233px',
+		'padding-top': '25.82px',
+	},
+	logoutButton: {
+		height: '28.89px',
+		width: '72.61px',
+		border: '1px solid #FFFFFF',
+		'border-radius': '100px',
+		opacity: '100%',
+		'background-color': 'Transparent',
+		color: '#FFFFFF',
+		'letter-spacing': '0.18px',
+		'line-height': '13px',
```

Comment:
The `opacity` and `line-height` css props are not necessary/used. Let me know if you'd like to chat about how I know that 😉 

Also, super minor nit: the `T` in `Transparent` doesn't need to be capitalized and the style for css pros is generally lowercase.